### DISCUSSION TITLE
VScode fails to find files installed in vcpkg include folder

### DISCUSSION THREAD

**Discussion by kmilo9999** [microsoft/vcpkg#33604] [Q&A]
Current environment:

- Windows 11
- VsCode 1.81.1
- C/C++ extension v1.17.5
- CMake Tools extension v1.15.31
- vcpkg package management program version 2023-08-09-9990a4c9026811a312cb2af78bf77f3d9d288416
- C/C++ compiler : msys64\\mingw64\\bin\\g++.exe
- C/C++ debugger: msys64\\mingw64\\bin\\gdb.exe

Hello all,

My vs code throws intellisense and compiling errors when I am including libraries installed using vcpkg

<img src="https://github.com/microsoft/vcpkg/assets/16551165/0b917507-6290-4f19-9ed1-4b55391e9471" width="700" height="500">

I ran the command  `vcpkg integrate install` 
This is my settings.json:

```
{
    "files.associations": {
        "string": "cpp"
    },
    "cmake.configureOnOpen": true,
    "cmake.useCMakePresets": "auto",    
    "cmake.configureSettings": {
        "DCMAKE_TOOLCHAIN_FILE": "C:/vcpkg/scripts/buildsystems/vcpkg.cmake"
    }
}
```
This is my c_cpp_properties.json:

```
{
    "configurations": [
        {
            "name": "Win32",
            "includePath": [
                "${workspaceFolder}/**",
                "${vcpkgRoot}/x64-windows/include"
            ],
            "defines": [
                "_DEBUG",
                "UNICODE",
                "_UNICODE"
            ],
            "windowsSdkVersion": "10.0.22000.0",
            "compilerPath": "cl.exe",
            "cStandard": "c17",
            "cppStandard": "c++17",
            "intelliSenseMode": "windows-msvc-x64",
            "configurationProvider": "ms-vscode.cmake-tools"
        }
    ],
    "version": 4
}
```

Building the project in cmake using the command line works. However, running the project from vs code fails due to the error showed above.
I have looked other discussions about the same topic, but none of their solutions work for me.

I appreciate any help.

**Comment by Osyotr** [comment]
`DCMAKE_TOOLCHAIN_FILE` -> `CMAKE_TOOLCHAIN_FILE`?

**Reply by kmilo9999** [reply]
I didnt work.

**Reply by dg0yt** [reply]
Did you even try from a clean build dir... CMAKE_TOOLCHAIN_FILE only works on the first run.

**Reply by kmilo9999** [reply]
Yes, I did that too. Multiple times. The debugger and launcher are just not picking up the include files. I've see other people having the same issue but no clear solution is given :(
